packages/web-config-server/src/apiV1/organisationUnit.js

```diff
@@ -21,7 +21,7 @@ export default class extends RouteHandler {
 
     // If this is a project entity, get entitiesWithAccess and set them on the entity
     if (this.entity.isProject()) {
-      const orgUnits = await this.entity.getNearestOrgUnitDescendants();
+      const orgUnits = await this.entity.getChildren(project.entity_hierarchy_id);
```

Comment:
This **_is_** actually a functionality change here... The previous `getNearestOrgUnitDescendants()` returns the nearest descendants of the valid 'orgUnit' types. I might have to double check this but I don't think we actually have any project with non-orgUnit entities as the immediate children of a project.

Switching to `getChildren()` here greatly reduces the entityHierarchyCache size as otherwise we end up caching the entire project hierarchy...